assembl/views/api2/content_source.py

```diff
@@ -22,11 +22,17 @@
 def fetch_posts(request):
     ctx = request.context
     csource = ctx._instance
-    force_restart = request.params.get('force_restart', False)
-    reimport = request.params.get('reimport', False)
-    reprocess = request.params.get('reprocess', False)
-    upper_bound = request.params.get('upper_limit', None)
-    lower_bound = request.params.get('lower_limit', None)
+
+    def request_parameter(key, default_value=None):
+        # when client POSTs data as JSON, in Pyramid data does not arrive in request.POST but in request.json_body
+        return request.json_body.get(key, default_value)
```

Comment:
I recall that the payload of an application/json comes in json_body, but everything in the `request.GET` and `request.POST` body come in the `request.params`.